### PR TITLE
Es6

### PR DESCRIPTION
**Pull request by OnlyUnknown**
- Added main contractor module file
- Added UI module file
- Added Transitions module file
- Added storage file
- Changed the old functions into ES6 syntax

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,112 @@
...
+
+I would like to thank my friends Yidne.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (1GORDON) [inline README.md]: Great job @OnlyUnknown 🤝

- [ ] Could you fix the MIT license link to make your readme file more professional?
- [ ] You may also remove the `NOTE` which shows details of creating the MIT license.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { contact, bookShelf, addNew } from './modules/Transisions.js';
+import UI from './modules/UI-Class.js';
```
Comment (1GORDON) [inline index.js:2]: Great job @OnlyUnknown 🤝

- [ ] As stated in the project requirements, kindly ensure the current date is displayed below the navigation bar. You can achieve this by using the [Luxon](https://github.com/moment/luxon/blob/master/docs/install.md) library which is added as a .js file to the project (in full or minified version). Then the module is imported `import { DateTime } from "luxon";`

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @OnlyUnknown ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @OnlyUnknown , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- Good commit messages. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**